### PR TITLE
make gui a command

### PR DESCRIPTION
**Pull request by pgunn**
This diff:
* Moves the current main GUI out of the library part of the code (nothing uses it as a library and it's a script). While doing so it gives it a better name
* Retires the ancient GUI that used to have that name to use_cases, which is more-or-less our junk drawer
* Adds a hashbang to the main GUI so people can launch it without saying python first
* Changes the install plumbing to make a "bin" subdirectory for caiman_data, which will be generally be a good place to put binaries of interest. The main GUI will live under ~/caiman_data/bin/
* Changes the GUI docs accordingly

### CODE REVIEW

**Comment by pgunn** [comment]
This is meant to resolve issue #700

**Comment by pgunn** [comment]
It also fixes a part of the docs that were not formatted right that I noticed while working on the diff

File: docs/source/GUI.rst [hunk 1]
```diff
@@ -13,23 +13,23 @@ how use it, first load and run either \* demo_OnACID_mesoscope.py \*
...
 Each of these demos will save a results file. You can then start the
-visual interface by running the following command from the base caiman
-folder (make sure you are within your caiman environment):
+visual interface by visiting the caiman_data folder, under the bin folder,
+and issuing the following command:
 
 ::
 
-   ipython caiman/gui/gui_pyqtgraph_layout.py
+   python caiman_gui.py
```
Comment (epnev) [inline docs/source/GUI.rst]: same question as earlier: `python` or `ipython`?
@pgunn

Comment (pgunn) [reply docs/source/GUI.rst]: I tested it with regular python and it behaved fine. It doesn't seem to depend on ipython.

**Pull request merged**